builds/fabric8-che/plugins/keycloak-plugin-shared/src/main/java/com/redhat/che/keycloak/shared/KeycloakSettings.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.che.keycloak.shared;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+
+import java.util.logging.Logger;
```

Comment:
no I don't really want you to change/test it now. I would say it would be use that time (or later) to use a filter (instead of valve) instead of changing java.util.logger to SLF4J